### ISSUE TITLE
Accessibility Issue

### ISSUE THREAD

**Issue by deremakif** [hackiftekhar/IQKeyboardManager#1964]
**Describe the bug**
There is a redundant button in the middle of the top toolbar. It might be related to the auto-correction or suggestions feature of the keyboard. I set the following features, and they work as expected:

toolbarPreviousBarButtonItemAccessibilityLabel, 
toolbarNextBarButtonItemAccessibilityLabel, and toolbarDoneBarButtonItemAccessibilityLabel

But I could not find any property to hide this button.

**To Reproduce**
1. Open the keyboard
2. Open accessibility inspector
3. You will see the button in the screenshots section.

**Expected behavior**
There is nothing in there. There should be a feature to set that accessibility label like: toolbarSuggestionBarButtonItemAccessibilityLabel

**Screenshots**

![Simulator Screen Shot - iPhone 14 Pro Max - 2023-06-15 at 19 25 07](https://github.com/hackiftekhar/IQKeyboardManager/assets/37242282/0ae7b61e-5d77-4a36-b0c8-55d54cb3cf3c)

### Versions
**Xcode:** 14.2
**Mac OS:** 12.5.1
**Simulator/Device:** Simulator & Device
**Simulator/Device Name:**  iPhone 14 Pro Max - 16.2
**Library Version:** 6.3.0

**Comment by halkbank-mimari** [comment]
We are using IQKeyboardManager, not IQKeyboardManagerSwift in our project for some reason. Can you please fix it for IQKeyboardManager too?

@hackiftekhar Thank you.
